doc/source/whatsnew/v0.20.0.txt

```diff
@@ -213,8 +213,7 @@ See :ref:`the docs here <io.pickle.compression>`
 
 Using an explicit compression type
 
-.. ipython:: python
```

Comment:
I'm not certain why this happened. The rest of the diff shows that the same identical line is added again, thus making no actual changes to the file